src/Core/Settings.cpp

```diff
@@ -2461,7 +2461,7 @@ Possible values:
 
 - default
 
- This is the equivalent of `hash` or `direct`, if possible (same as `direct,hash`)
+ This is the equivalent of `hash`, `parallel_hash` or `direct`, if possible (same as `direct,parallel_hash,hash`)
```

Comment:
The join algorithms in the first part of the sentence are in the wrong order.

Maybe just

```
Same as `direct,parallel_hash,hash`, i.e. try using `direct`, `parallel_hash` or `hash` in this order and if applicable each.
```

?